packages/storage-layer-torus/src/TorusStorageLayer.ts

```diff
@@ -199,8 +200,7 @@ class TorusStorageLayer implements IStorageLayer {
 
     const hash = keccak256(Buffer.from(stringify(setTKeyStore), "utf8"));
     if (privKey) {
-      const ecCurve = keyTypeToCurve(keyType);
-      const signKeyPair = ecCurve.keyFromPrivate(privKey.toBuffer());
+      const signKeyPair = getPrivateKeyForSigning(privKey, keyType);
```

Comment:
We need to check this import has the same behaviour as the replaced code.

Reply:
yes, i checked, old code here wont work now because we need to use secp key for metadata signing same as we are doing for encryption.

Reply:
I see. Ok